packages/app/src/cli/utilities/app-conf-store.test.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  getConfigStoreForAccountInfoStatus,
+  getCachedAccountInfo,
+  setCachedAccountInfo,
+  clearCachedAccountInfo,
+} from './app-conf-store.js'
+import {AccountInfo} from '../services/context/partner-account-info.js'
+import {vi, describe, test, expect, beforeEach, afterEach} from 'vitest'
+import {LocalStorage} from '@shopify/cli-kit/node/local-storage'
+import {inTemporaryDirectory} from '@shopify/cli-kit/node/fs'
+import {joinPath} from '@shopify/cli-kit/node/path'
+
+describe('app-conf-store', () => {
+  let tempDir: string
+
+  beforeEach(async () => {
+    vi.useFakeTimers()
+    vi.setSystemTime(new Date('2023-01-01'))
+    await inTemporaryDirectory(async (dir) => {
+      tempDir = dir
+      process.env.XDG_CONFIG_HOME = joinPath(tempDir, '.config')
+    })
+  })
+
+  afterEach(() => {
+    vi.useRealTimers()
+    delete process.env.XDG_CONFIG_HOME
```

Comment:
this may prove flaky -- any other options? can we inject preference (config folder maybe? or process.env?) into the config functions?